Comparing source compatibility of /Users/bluefire/git/uima-uimaj/target/checkout/uimaj-core/target/uima-core.jar against /Users/bluefire/.m2/repository/org/apache/uima/uimaj-core/3.3.0/uimaj-core-3.3.0.jar
WARNING: You have ignored certain classes, i.e. superclasses and interfaces that could not be found on the classpath are ignored. Hence changes caused by these superclasses and interfaces are not reflected in the output.
***  MODIFIED CLASS: PUBLIC org.apache.uima.cas.CASRuntimeException  (compatible)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String JCAS_TYPE_NOT_IN_CAS_REGISTRY
